src/middlewared/middlewared/plugins/pool_/dataset_encryption_info.py

```diff
@@ -228,8 +228,7 @@ def path_in_locked_datasets(self, tls, path):
         # WARNING: _EXTREMELY_ hot code path. Do not add more
         # things here unless you fully understand the side-effects.
         if path.startswith('/dev/zvol/'):
-            # 10 comes from len("/dev/zvol/")
-            path = path[10:].replace('+', ' ')
+            path = path.removeprefix('/dev/zvol/').replace('+', ' ')
```

Comment:
Like I said in previous PR, this is a micro-optimization so switching to using `removeprefix` from slicing isn't necessary. The code is perfectly readable as-is.